api/views.py

```
@@ -0,0 +1,104 @@
+            'encounters': EncounterSerializer(enc, many=True).data,
+            'labs': LabResultSerializer(labs, many=True).data,
+            'medications': MedicationOrderSerializer(meds, many=True).data,
+            'ai_summaries': AISummarySerializer(summaries, many=True).data
+        })
+
+
+class EncounterViewSet(viewsets.ModelViewSet):
+    queryset = Encounter.objects.all().order_by('-occurred_at')
+    serializer_class = EncounterSerializer
+
+    def perform_create(self, serializer):
```

Ruff (0.12.2)

Missing return type annotation for public function `perform_create`

Add return type annotation: `None`

(ANN201)